### ISSUE TITLE
Shadow under Shrine app bar is darker than YouTube app bar shadow

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#10238]
Brought up by someone on Twitter. Here's a screenshot:
![screenshot_20170520-233414](https://cloud.githubusercontent.com/assets/5479/26281735/2eb4a7da-3db5-11e7-83fb-ec3811a1f559.png)

**Comment by Hixie** [comment]
cc @jason-simmons

**Comment by sethladd** [comment]
Actually, this may be a widget thing, not an engine thing? Is this controllable by settings we applied to the Shrine app, or all apps?

**Comment by HansMuller** [comment]
The line below the shrine app bar is just a divider when the content hasn't been scrolled, and a divider + an elevation=1 material shadow when it  has. I suppose we could make it look more like the youtube appbar by leaving out the divider when the appbar is elevated. On the other hand, the goal was to look like the (internal) mock-ups for this app.

**Comment by sethladd** [comment]
Thanks @HansMuller ! Will leave this open for a little while longer, to see what other Google apps do here.

I also found https://material.io/guidelines/layout/structure.html#structure-app-bar which seems to show a darker shadow under the app bar (similar to what we have now in shrine).

Also, this: https://material.io/guidelines/components/menus.html

So, actually, going to close this as "works as intended", as far as we can tell.

**Comment by asarazan** [comment]
Hey folks,

I just created a blank android application in studio and grabbed the default shadow settings and they look more like youtube than flutter. With shadows being one of the most important components of Material, I feel like it's important to get them pixel perfect to the host platform with zero config.

Screenshot:
![untitled](https://cloud.githubusercontent.com/assets/542872/26361938/5cfb61f0-3f91-11e7-9001-2ea0d3308844.png)

**Comment by sethladd** [comment]
Thanks for the screenshot! Now, the question is, what does Flutter out-of-the-box look like? :)

Here's what I found:
![screen shot 2017-05-23 at 8 30 17 am](https://cloud.githubusercontent.com/assets/5479/26362203/1979923e-3f92-11e7-91c9-645bd04d6648.png)

**Comment by asarazan** [comment]
Yeah so it really does seem like the biggest problem here is the line not disappearing when elevation happens. That makes sense.

**Comment by sethladd** [comment]
Reopening to do a bit more research.

**Comment by sethladd** [comment]
@asarazan what is the exact color of the body of the default android app that you shared above? Thanks!

**Comment by Hixie** [comment]
I believe, though I could be wrong, that Flutter more closely matches what the Material team would like us to do than do the other apps mentioned here.

cc @willlarche for confirmation, or, if I am wrong, for more detailed instructions on how we should be drawing our shadows.
